pkg/cassandra/config/schema.go

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"bytes"
+	"embed"
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"text/template"
+
+	"github.com/jaegertracing/jaeger/pkg/cassandra"
+)
+
+//go:embed v004-go-tmpl.cql.tmpl
+//go:embed v004-go-tmpl-test.cql.tmpl
+var schemaFile embed.FS
+
+type TemplateParams struct {
+	Schema
+	// Replication is the replication strategy used
+	Replication string `mapstructure:"replication" valid:"optional"`
+	// CompactionWindowSize is the numberical part of CompactionWindow. Extracted from CompactionWindow
+	CompactionWindowSize int `mapstructure:"compaction_window_size" valid:"optional"`
+	// CompactionWindowUnit is the time unit of the CompactionWindow. Extracted from CompactionWindow
+	CompactionWindowUnit string `mapstructure:"compaction_window_unit" valid:"optional"`
+}
+
+func DefaultParams() TemplateParams {
```

Comment:
not needed. There is DefaultConfig function somewhere, it should populate these settings there.

Reply:
Removed. Thanks